catalog/core/catalog-core-api-impl/src/main/java/ddf/catalog/data/impl/MetacardImpl.java

```diff
@@ -843,15 +858,60 @@ private void readObject(ObjectInputStream stream) throws IOException, ClassNotFo
 
     }
 
+    @Override
+    public boolean equals(Object o) {
```

Comment:
Did we need this for something specifically? I think before we almost updated it but didnt' because we cant' guarantee that all metacard implementations will be using our `MetacardImpl` as its base, so using equals to compare two metacards is not deterministic.